### PR TITLE
Events route tests

### PR DESCRIPTION
**Pull request by Astatium5**

### CODE REVIEW

File: src/api/events.py [hunk 1]
```diff
@@ -59,10 +60,16 @@ def create_event():
...
-        data["date_time"] = dateutil.parser.parse(data["date_time"])
+        try:
+            data["date_time"] = dateutil.parser.parse(data["date_time"])
+        except ParserError:
+            raise BadRequest()
 
     if data["end_date_time"]:
-        data["end_date_time"] = dateutil.parser.parse(data["end_date_time"])
+        try:
+            data["end_date_time"] = dateutil.parser.parse(data["end_date_time"])
```
[the remark this answers is not in the document]
Comment (AdamPellot) [reply src/api/events.py:70]: Fix this line too long issue

**Review by AdamPellot — CHANGES_REQUESTED** [review]
Everything looks good just fix the "line too long" issue the linter caught.

File: src/api/events.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (Astatium5) [reply src/api/events.py:70]: I am not sure how to fix it in this case.

**Review by AdamPellot — APPROVED** [review]
Nice job

**Pull request merged**